app/types/CollectionCard.ts

```diff
@@ -0,0 +1,12 @@
+export interface CollectionCardData {
```

Comment:
True.. think it would be more clear but I was just following the pattern already in place with the following types: FeaturedItemData, ExploreFurtherData, and FeaturedContentData . If we add ...Type to the new types, I think we should also add them the existing ones - do you agree?

Reply:
also wondering if we should have `props` directory in the types folder

Reply:
Try the `props` directory? It's a reversible small update so you can always try it to see if it's helpful.

Personally, I like to keep the prop type/interface in the same file as the component for readability. It's the convention in the DS and maybe other projects.

Reply:
I agree we should add ...Type to everything. `props` directory is up to you if you think it'd be good @avertrees